sources/react-app/src/shared/hooks.js

```diff
@@ -107,10 +340,12 @@ export function useUrlSearchQueryFetchResource() {
             'bool': {
               'filter': [
                 { 'bool': { 'should': contentTypes.map(id => ({ 'match': { 'content-type': id } })) } },
-                { 'multi_match': { 'query': query, 'fields': fields } }
+                { 'multi_match': { 'query': query, 'type': "phrase_prefix"} }
```

Comment:
Single quote

Reply:
Is the query matching all relevant fields?

Reply:
Yes it is, I've tried searching by title, content, author, category, etc..